sdk/servicebus/service-bus/test/utils/managementUtils.ts

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+  QueueOptions,
+  TopicOptions,
+  SubscriptionOptions,
+  ServiceBusAtomManagementClient,
+  delay
+} from "../../src";
+
+import { EnvVarKeys, getEnvVars } from "./envVarUtils";
+
+const env = getEnvVars();
+let client: ServiceBusAtomManagementClient;
+
+async function getManagementClient() {
+  if (client == undefined) {
+    client = await new ServiceBusAtomManagementClient(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]);
+  }
+  return client;
+}
+
+/**
+ * Utility to apply retries to `create` management operations.
+ * Will ignore if given entity is already existing.
+ * @param createEntityOperation
+ * @param checkIfEntityExistsOperation
+ */
+async function retryCreateManagementEntityOperation(
+  createEntityOperation: any,
+  checkIfEntityExistsOperation: any
+) {
+  const retryAttempts = 5;
+  const retryDelayInMs = 1000;
+
+  let succeeded: boolean = false;
+  let count = 0;
+  while (count < retryAttempts) {
+    try {
+      const entityExists = await checkIfEntityExistsOperation();
+      if (entityExists) {
+        succeeded = true;
+        break;
+      }
+      await createEntityOperation();
+    } catch (err) {
+      // Ignore error and wait before retrying
```

Comment:
Is there a way to log the fact that we retried? It might be interesting to see in the logs how often we run into this issue.

Reply:
Actually we had decided to not custom log things as there might be tooling that rely on parsing standard test output logs.
But I agree and think this is a great point, we could do this when we have some kind of metrics infrastructure support available.

Reply:
I see. I wonder if we could use the debug package here. But this isn't necessary for this PR.